src/pixelator/plot/__init__.py

```diff
@@ -935,6 +939,165 @@ def plot_colocalization_diff_volcano(
     return fig, ax
 
 
+def __plot_joint_distribution(data, x, y, **kargs):
+    g = sns.JointGrid(data, x=x, y=y)
+    g.plot_marginals(sns.kdeplot)
+    g.plot_joint(
```

Comment:
Style suggestion: Is it possible to remove the axes of the marginal plots, and use fill color instead of stroke color?

Reply:
Done.